docs/admin-ports.md

```diff
@@ -38,7 +38,6 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
- - 62500: Networking API
```

Comment:
I don't have a good answer for that. I could copy the doc and keep this port, but without a process to sync that version with this one, it'll just go out of date.